blueno/elasticsearch.py

```diff
@@ -91,6 +93,92 @@ class Index:
         name = TRAINING_JOBS
 
 
+class ValidationJob(elasticsearch_dsl.Document):
+    """
+    Object for validation data.
+    TODO: Can this be merged with TrainingJob, with a common
+        parent object?
+    """
+    id = elasticsearch_dsl.Integer()
```

Comment:
this shadows a built in function — you might want to change it to id_ or something